langchain/experimental/autonomous_agents/autogpt/agent.py

```diff
@@ -37,15 +37,20 @@ def __init__(
         output_parser: BaseAutoGPTOutputParser,
         tools: List[BaseTool],
         feedback_tool: Optional[HumanInputRun] = None,
+        chat_history_persistence_file_path: Optional[str] = None,
     ):
         self.ai_name = ai_name
         self.memory = memory
-        self.full_message_history: List[BaseMessage] = []
         self.next_action_count = 0
         self.chain = chain
         self.output_parser = output_parser
         self.tools = tools
         self.feedback_tool = feedback_tool
+        self.chat_history_memory = (
```

Comment:
lets just change this so users can pass in a memory object

Reply:
make sense :) will do

Reply:
Done + updated the relevant doc